rosalind_ex/change_problem.py

```diff
@@ -0,0 +1,36 @@
+
+
+#input: integer money and array coins of d positive integers which are the denominations
+#output: the min nimber of coins w demoninations coins that changes money
+
+def dp_change(money, coins):
+	MinNumCoins = [0]
+	#print(MinNumCoins)
+
+	for m in range(1, money + 1):
+		MinNumCoins.append(1000) #said to go to infitiny, I arbitrarily picked this and it worked, but what is the real rule for how big I need to go?
```

Comment:
You can use sys.maxsize for this (that's the maximum value of a integer in Python).

Reply:
Do you know why this is is necessary?